docs/tutorial-usage/make-a-bet.md

```diff
@@ -4,32 +4,32 @@ sidebar_position: 3
 
-Fill out the **New Bet** form to create and fund a bet. Let's take a look of each part of an example bet:
+Fill out the **New Bet** form to create and fund a bet. Let's take a look at each section of this example:
 
 ![New Bet](/img/newbet.png)
 
 ## Conditions
-
-This what the Judge should use to determine a winner. It is a string of text that could be anything the Maker feels like betting on. It is recommended that the Maker should make clear terms for thier bet conditions to avoid confusion and cancelled bets. 
+ 
+The conditions is what the Judge should review to determine a winner. It is a string of text which could be anything the Maker feels like creating a bet on. To avoid confusion and cancellations, the Maker should clearly define the terms of their bet.
 
 ## Maker
 
-The person making the bet. The address is autopopulated with the logged in user's wallet, but it could technically be anyone. This person will also be placing thier own money into the bet as a wager.
+The Maker is the person making the bet. The address is autopopulated witht the wallet the user logged in with, but it could technically be anyone. This individual will also be placing their own funds into the wager based on the conditions specified.
 
 ## Taker
 
-This is the person on the other side of the bet. Confirm that this has been inputted correctly, if the address or username does not map to an EVM address the form will be blocked from being completed, but if the address or username is wrongly inputed to someone you do not know, then you will have to cancel the bet and try again. 
+This is the person on the other side of the bet. Ensure that their address or username is entered correctly. If it does not correspond to an EVM address, the form will be blocked from completion. If the address or username is incorrectly entered from someone you don’t know, you will need to cancel the bet and try again.
 
 ## Judge
 
-This is the person that decides who wins the bet. It is important to choose the Judge carefully and make sure this is someone that both parties can trust. The conditions of the bet are technically arbitrary for how the bet is resolved, it is entirely up to the descrition of the Judge. The Judge recieves no financial compensation for thier role, this arrangement is entirely trust based. 
+This is the person who decides the outcome of the bet. It’s important to choose the Judge carefully, ensuring they are someone both parties can trust. The bet’s resolution is entirely up to the Judge’s discretion, as the conditions are technically arbitrary. The Judge does not receive any financial compensation; this arrangement is based entirely on trust.
 
-The Judge may be the Maker or the Taker for simplicity. This arrangement is not recommended for high stakes wagers or bets with argumentative people. 
+For simplicity, the Judge can be either the Maker or the Taker, though this setup is not advisable for high-stakes wagers or when dealing with oposing opinions.
```

Comment:
i still prefer 'argumentative people' over oposing opinions at the end of this sentance